### ISSUE TITLE
QASC dataset: answerKey missing in test split 

### ISSUE THREAD

**Issue by rajcscw** [tensorflow/datasets#2450]
**Short description**
In [QASC](https://www.tensorflow.org/datasets/catalog/qasc?hl=cs) dataset, answerKey field is missing in the test samples.

**Environment information**
* Operating System: Ubuntu
* Python version: 3.7
* `tensorflow-datasets`/`tfds-nightly` version:  3.2.1
* `tensorflow`/`tf-nightly` version:  3.2.1.dev202009090105

**Reproduction instructions**

```python
import tensorflow_datasets as tfds
import tensorflow as tf

tf.enable_eager_execution()
ds = tfds.load('qasc', split=split, shuffle_files=True)

for datapoint in ds:
    facts = [datapoint["fact1"].numpy().decode("utf-8"), datapoint["fact2"].numpy().decode("utf-8")]
    question = datapoint["question"].numpy().decode("utf-8")
    choices = {label.decode("utf-8"): text.decode("utf-8")
                for label, text in zip(datapoint["choices"]["label"].numpy(),
                                        datapoint["choices"]["text"].numpy())}
    answer = datapoint["answerKey"].numpy().decode("utf-8")
    print(answer)

```

**Comment by Conchylicultor** [comment]
Does the dataset actually publish a test set ? Otherwise, you'll have to manually create your test set with the https://www.tensorflow.org/datasets/splits API

**Comment by rajcscw** [comment]
Yes, seems like the test split is available for this dataset https://www.tensorflow.org/datasets/catalog/qasc, but the labels are missing. Not sure, if this is intentional.

**Comment by Conchylicultor** [comment]
The issue could have been rename. Test split exists but label are missing.

Are the labels available in the original dataset ? Some datasets have private labels, and require users to upload result on their leaderboard (e.g. imagenet has public train and val set, but private testing set).

As https://allenai.org/data/qasc has a leaderboard, I think the labels are private to avoid cheating.

**Comment by rajcscw** [comment]
Ah, you are right. Totally makes sense. Maybe, the documentation of the dataset could have some explanation regarding this.

**Comment by Conchylicultor** [comment]
Yes, probably, don't hesitate to send a PR for this.
